tc/sgx/trusted_worker_manager/enclave/work_order_data_handler.h

```diff
@@ -31,28 +31,71 @@ namespace tcf {
         public:
             WorkOrderDataHandler() {}
 
-            explicit WorkOrderDataHandler(const WorkOrderData& wo_data, std::string encrypted_session_key) {
+            explicit WorkOrderDataHandler(ByteArray session_key,
+                                          ByteArray session_key_iv) {
+                this->session_key = session_key;
+                this->session_key_iv = session_key_iv;
```

Comment:
Can we initializing other member variables to default empty values .

Reply:
other member variables are initialized to their default values at the declaration place. 
Do you suggest to initialize in the constructor as well?

Reply:
Yes. adding in constructor is better than at declaration place.